docs/guide-bedtools.md

```diff
@@ -0,0 +1,101 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.3
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Bioframe for bedtools users
+
+If you want to work on `gtf` files, you do not need to turn them into bed files,
```

Comment:
this info on GTF files should migrate to the `Reading genomic dataframes[](https://bioframe--198.org.readthedocs.build/en/198/guide-io.html#reading-genomic-dataframes)` section

Reply:
Why not both ?

That way if people come from `bedtools` they directly see the info instead of having to click another tab ?

Reply:
well gtfs vs. bed files is not really about bedtools per se... perhaps @nvictus has another suggestion?

Reply:
The rationale is to do some handholding for the `bedtools` folks so that they have everything in a single page, I also added a link to the `io.fileops` module and the read_table operation, as well as `to_bed`

Reply:
I like the expanded intro! Here are a few more suggestions (sorry some hyperlinks/special text got dropped, but those were nice to have as well!)

bioframe is built around the analysis of genomic intervals as a pandas [DataFrame](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html) in memory, rather than working with tab-delimited text files saved on disk. Bioframe supports reading a number of standard genomics text file formats  via read_table, including BED files (see schemas {add hyperlink}), which will load them as pandas DataFrames. For example, with  gtf files, you do not need to turn them into bed files, you can directly read them into pandas (with e.g. [gtfparse](https://github.com/openvax/gtfparse/tree/master)). For gtfs, it is often convenient to rename the seqname column into chrom, the default column name used in bioframe. Any DataFrame object with 'chrom', 'start', and 'end' columns will support the genomic interval operations in bioframe {add link}. Finally, if needed, bioframe provides a convenience function to write the back to a bed file using to_bed {add link}.

Reply:
Done :)

There is a link missing to `to_bed`, but it is missing a page in the readthedoc, I'll add it in a followup PR